platform-coop-toolkit/coop-library-framework
# Reorganize fields (resolve #132)

## Description

Reorganizes resource meta fields as discussed.

## Steps to test

1. Edit a resource.

**Expected behavior:** Field layout matches [this spec](https://docs.google.com/spreadsheets/d/1cfGei_Rw9PGesvxJa6IQmj8PT-4xeXNbdiAH_82mOa8/edit).

## Additional information

Not applicable.

## Related issues

- Resolves #132.
